release_notes/ocp_3_5_release_notes.adoc

```diff
@@ -0,0 +1,1683 @@
+====  Qualification of External Dynamic Provisioner Interface and Thrird-party PV
+
+In {product-title} 3.5, there is now the qualification of the Kubernetes
+interface for an external dynamic provisioner so that Red Hat can support a
+customer using a third-party storage solution such as
+xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#available-dynamically-provisioned-plug-ins[NetApp
+Trident].
+
+There is a concept of _in-tree_ and _out-of-tree_ with
+Kubernetes storage. Out-of-Tree means that it is not in the Kubernetes source tree
+and does not ship in Kubernetes or {product-title}. The ability is provided post-installation.
+Many of the third-party storage vendors gravitate towards out-of-tree because it
+allows them to ship on their own schedule and own the distribution of their
+code.
+
+See
+xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#available-dynamically-provisioned-plug-ins[Available
+Dynamically Provisioned Plug-ins] for more information.
+
+[[ocp-35-dynamic-provisioner-for-azure-block-storage]]
+==== Dynamic Provisioner for Azure Block Storage
+
+Dynamic provisioning is now available for Azure block storage. Just like AWS and
+GCE, you declare the Azure cloud provider in the *_cloud-config_* file, and then
+create `StorageClasses` with the Azure block storage options and connection
+information.
+
+.Configure the Cloud Provider for Azure
+----
+apiServerArguments:
+    cloud-provider:
+      - "azure"
+      cloud-config:
+        - "/etc/azure/azure.conf"
+  controllerArguments:
+    cloud-provider:
+      - "azure"
+      cloud-config:
+        - "/etc/azure/azure.conf"
+----
+
+.Example StorageClass
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: slow
+provisioner: kubernetes.io/azure-disk
+parameters:
+  skuName: Standard_LRS
+  location: eastus
+  storageAccount: azure_storage_account_name
+----
+
+See
+xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#azure-disk[Dynamic
+Provisioning and Creating Storage Classes] for more information.
+
+[[ocp-35-scale]]
+=== Scale
+
+[[ocp-35-scalability-enhancements]]
+==== Scalability Enhancements for Metrics
+
+With {product-title} 3.5, the default value of the `METRICS_RESOLUTION`
+parameter is now `30` (seconds). This change was introduced to better match the
+cAdvisor housekeeping interval of 30 seconds
+link:https://bugzilla.redhat.com/show_bug.cgi?id=1421834[(BZ#1421834)].
+
+Increasing the `METRICS_RESOLUTION` interval helped achieve better results in
+relation to how many pods can be monitored by one set of metrics pods. In
+{product-title} 3.5, tests showed that OpenShift metrics collection was stable for
+test cases up to 25,000 monitored pods in a {product-title} cluster.
+
+See
+xref:../install_config/cluster_metrics.adoc#metrics-recommendations-for-OCP-version-35[Enabling
+Cluster Metrics] for more information.
```

Comment:
This link will change once https://github.com/openshift/openshift-docs/pull/3881 lands